test/unit/import_playbooks_test.rb

```diff
@@ -0,0 +1,57 @@
+require 'test_plugin_helper'
+
+class ImportPlaybooksTest < ActiveSupport::TestCase
+  let(:ansible_proxy) { FactoryBot.create(:smart_proxy, :with_ansible) }
+  let(:proxy_api) { ::ProxyAPI::Ansible.new(url: ansible_proxy.url) }
+  let(:playbooks_names) { ['xprazak2.forklift_collection.collect_debug_draft.yml'] }
+  subject { ::ForemanAnsible::PlaybooksImporter.new(ansible_proxy) }
+
+  setup do
+    subject.stubs(:proxy_api).returns(proxy_api)
+    let(:sample_playbooks) { JSON.parse(File.read(ansible_fixture_file('sample_playbooks.json'))) }
+    let(:playbook1_output) { sample_playbooks['1'] }
+    let(:playbook2_output) { sample_playbooks['2'] }
```

Comment:
`let` is lazy_loaded (initialized on the first use) so it is weird to put it in the `setup` block. it should be outside of the block with the other `let` clauses